packages/docusaurus-theme-translations/locales/vi/theme-common.json

```diff
@@ -3,32 +3,32 @@
   "theme.BackToTopButton.buttonAriaLabel": "Trở lại đầu trang",
   "theme.CodeBlock.copied": "Đã sao chép",
   "theme.CodeBlock.copy": "Sao chép",
-  "theme.CodeBlock.copyButtonAriaLabel": "Sao chép code vào bộ nhớ tạm",
+  "theme.CodeBlock.copyButtonAriaLabel": "Sao chép mã vào bộ nhớ tạm",
   "theme.CodeBlock.wordWrapToggle": "Chuyển đổi văn bản xuống dòng",
   "theme.DocSidebarItem.collapseCategoryAriaLabel": "Thu gọn danh mục '{label}'",
   "theme.DocSidebarItem.expandCategoryAriaLabel": "Xem thêm danh mục '{label}'",
-  "theme.ErrorPageContent.title": "Trang này đã bị lỗi.",
+  "theme.ErrorPageContent.title": "Trang này đã gặp lỗi.",
   "theme.ErrorPageContent.tryAgain": "Thử lại",
   "theme.NavBar.navAriaLabel": "Thanh điều hướng",
-  "theme.NotFound.p1": "Chúng tôi không thể tìm thấy những gì bạn đang tìm kiếm.",
-  "theme.NotFound.p2": "Vui lòng liên hệ với trang web đã dẫn bạn tới đây và thông báo cho họ biết rằng đường dẫn này bị hỏng.",
+  "theme.NotFound.p1": "Chúng tôi không thể tìm thấy nội dung bạn đang tìm kiếm.",
+  "theme.NotFound.p2": "Vui lòng liên hệ với trang web đã đưa bạn đến đây và thông báo rằng đường dẫn này bị lỗi.",
   "theme.NotFound.title": "Không tìm thấy trang",
   "theme.TOCCollapsible.toggleButtonLabel": "Trên trang này",
-  "theme.admonition.caution": "cẩn thận",
-  "theme.admonition.danger": "cảnh báo",
-  "theme.admonition.info": "thông tin",
-  "theme.admonition.note": "ghi chú",
-  "theme.admonition.tip": "mẹo",
-  "theme.admonition.warning": "warning",
+  "theme.admonition.caution": "Cẩn thận",
+  "theme.admonition.danger": "Cảnh báo",
+  "theme.admonition.info": "Thông tin",
+  "theme.admonition.note": "Ghi chú",
+  "theme.admonition.tip": "Mẹo",
+  "theme.admonition.warning": "Cảnh báo",
```

Comment:
Do we need to add uppercase here? 

The other languages do not have this and handle uppercasing with CSS afaik.

Reply:
If you using text-transform: uppercase; I think we don't need update them.